packages/api/src/brevo-contact/brevo-contact.resolver.ts

```diff
@@ -45,15 +59,31 @@ export function createBrevoContactResolver({ BrevoContact }: { BrevoContact: Typ
             @Args("id", { type: () => Int }) id: number,
             @Args("input", { type: () => BrevoContactUpdateInput }) input: BrevoContactUpdateInput,
         ): Promise<BrevoContactInterface> {
-            return this.brevoContactApiService.updateContact(id, input);
+            return this.brevoContactsApiService.updateContact(id, input);
         }
 
         @Mutation(() => Boolean)
         async deleteBrevoContact(@Args("id", { type: () => Int }) id: number): Promise<boolean> {
-            return this.brevoContactApiService.deleteContact(id);
+            return this.brevoContactsApiService.deleteContact(id);
         }
 
-        // TODO: subscribe to newsletter
+        @Mutation(() => SubscribeResponse)
+        @PublicApi()
```

Comment:
```suggestion
```
I would remove the PublicApi(). It can be called by a BFF (in the next api)

Reply:
removed in [86f6f4b](https://github.com/vivid-planet/comet-brevo-module/pull/7/commits/86f6f4bb0f858580e77546f9bb44b945f375636c)